annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastFieldsResolver.java

```diff
@@ -0,0 +1,60 @@
+package com.exactpro.epfast.annotation.processing;
+
+import com.exactpro.epfast.annotations.FastField;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.Elements;
+import java.util.*;
+
+class FastFieldsResolver {
+    private ArrayList<FastTypeElement> cache = new ArrayList<>();
+
+    private Elements elements;
+
+    public FastFieldsResolver(Elements elements) {
+        this.elements = elements;
+    }
+
+    private boolean isOverrideMethod(List<FastFieldElement> fastFields, String methodName) {
```

Comment:
change to `isOverriddenIn(FastTypeElement fastType, FastFieldElement fastField)`